README.md

```diff
@@ -6,30 +6,31 @@ HSTS Parser is a simple tool to parse Firefox and Chrome's HSTS databases into a
 
 ## Installation
 
-Installing HSTS Parser is easy! Just run the below commands to install everything you need!
+HSTS Parser can be installed via pip, or with [Poetry](https://python-poetry.org/).
 
-### Linux
+### From PyPi
+```bash
+pip install hstsparser
+```
 
+### Poetry (Linux)
 ```bash
 curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
-poetry install
+poetry install --no-dev
 ```
 
-### Windows
-
+### Poetry (Windows)
 ```powershell
 (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
-poetry install
+poetry install --no-dev
 ```
 
 Alternatively, if you're using Windows, you can use the executables on the [releases page](https://github.com/thebeanogamer/hstsparser/releases/latest) to not need to install anything at all.
 
 ## Usage
 
-All of the below documentation is written for the Python version rather than the standalone, but the commands are the same, just replacing `poetry run python hstsparser.py` with the name of the executable.
```

Comment:
I'm a bit iffy on removing this line, but if it did stay it would obviously need to be expanded.